validation/src/main/scala/hmda/validation/engine/ValidationError.scala

```diff
@@ -7,7 +7,8 @@ case object Quality extends ValidationErrorType
 case object Macro extends ValidationErrorType
 
 // errorID = Loan ID (LAR) or Agency Code + Respondent ID (TS)
-case class ValidationError(errorId: String, name: String, errorType: ValidationErrorType)
+case class ValidationError(errorId: String, metaData: ValidationErrorMetaData, errorType: ValidationErrorType)
+case class ValidationErrorMetaData(name: String)
```

Comment:
Would it be possible (without a large time commitment) to remove `ValidationErrorMetaData` from this PR? Since that class won't be used until a future PR, I think it doesn't make sense to code review and merge it now.

I don't want to waste your time doing git stuff, but I also want to wait and evaluate `ValidationErrorMetaData` in context, since it's a big change in a confusing part of the code.